administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -63,35 +180,39 @@ function triggerSave() {
 		height: 1500px;
 		border: 0 !important;
 	}
+
+	.reference-side h3 {
+		float: left;
+		width: 50%;
+	}
 ');
 ?>
-<button id="toogle-left-panel">Show/Hide Reference (PoC)</button>
+<button id="toogle-left-panel" class="btn btn-small">Show/Hide Reference (PoC)</button>
 
-<form action="<?php echo JRoute::_('index.php?option=com_associations&view=association'); ?>"
- method="post" name="adminForm" id="item-form" class="form-validate">
+<form action="<?php echo JRoute::_(JFactory::getURI()->toString()); ?>"
```

Comment:
it's better to add the all URL parameters inside JRoute, like all other views.
We need always to be consistent with the rest of the code in the core.